### ISSUE TITLE
ERR spawn npm ENOENT on Windows 10.

### ISSUE THREAD

**Issue by ex-jedi** [wesbos/eslint-config-wesbos#73]
Just started getting the following error when attempting a local install:
```
install-peerdeps v3.0.0
ERR spawn npm ENOENT
```
It's been working for ages just fine. Any ideas what this could be?

Ta

**Comment by Asjas** [comment]
Hi, @ex-jedi 

What version of Node.js and NPM are you using?

There are some issues with `npm 7.x`, you can try to use `npm 6.x` and see if it works. 🙂

**Comment by ex-jedi** [comment]
Thanks @Asjas.

 I'm on NPM 6.14.10 & Node 14.15.4.

**Comment by Asjas** [comment]
Is there more information you can provide - such as what terminal you are using and what command you are using? 🙂

**Comment by Asjas** [comment]
Okay, Just checked the `install-peerdeps` Github repo and there is a reported issue with the `v3.0.0` release that happened 4 days ago. Other Windows 10 users are reporting the same issue. [#123](https://github.com/nathanhleung/install-peerdeps/issues/123
)

I'd suggest manually copying the [npm packages](https://github.com/wesbos/eslint-config-wesbos/blob/master/package.json#L30) to your own `package.json` file and then install them with `npm i` 🙂

**Comment by ex-jedi** [comment]
Thanks @Asjas that's fixed it! 👍🏾👍🏾
